pkg/integration/tests/commit/staged.go

```diff
@@ -53,8 +53,6 @@ var Staged = NewIntegrationTest(NewIntegrationTestArgs{
 				Contains(commitMessage),
 			)
 
-		t.Views().StagingSecondary().IsFocused()
-
-		// TODO: assert that the staging panel has been refreshed (it currently does not get correctly refreshed)
```

Comment:
we should add an assertion here that the staging panel has been refreshed (there are examples of asserting on a view's contents in various tests so I'd take a a look at those)